lisa/tests/kernel/scheduler/sanity.py

```diff
@@ -25,7 +26,7 @@ class CapacitySanity(TestBundle):
     A class for making sure capacity values make sense on a given target
 
     :param capacity_work: A description of the amount of work done on the
-      target, per capacity value ({capacity : work})
+      target, per capacity value ({capacity : [work_on_cpu0, work_on_cpu1, ...]})
```

Comment:
AFAICT when creating `capa_work` in `_from_testenv()`, we have no guarantee on the CPU iteration order. So instead we should use `{capacity : {cpu : work_on_cpu}}` and we can do the test on

```python
capacity_work =  {
    capa: min(cpus.values())
    for capa, cpus in self.capacity_work.items()
}
```

And then the test will be truly obvious.

Reply:
That's right, I left the CPU ID out since it was not used anywhere for the actual computation, and having the list of values can still be useful to check wildly wrong values (even without the CPU ID).